src/Commands/AdminCommand.php

```diff
@@ -50,6 +50,11 @@ public function handle()
             $this->option('create')
         );
 
+        // the user not returned
+        if (!$user) {
+            exit;
+        }
```

Comment:
Sorry to bother you again, but what do the above lines do/why are they necessary now?

Reply:
if the password doesnot match the a null `$user` is returned, hence if `!$user` then exit.

Reply:
Ah yeah, got it